src/init.cpp

```diff
@@ -368,6 +368,7 @@ std::string HelpMessage(HelpMessageMode mode)
     strUsage += HelpMessageOpt("-sysperms", _("Create new files with system default permissions, instead of umask 077 (only effective with disabled wallet functionality)"));
 #endif
     strUsage += HelpMessageOpt("-txindex", strprintf(_("Maintain a full transaction index, used by the getrawtransaction rpc call (default: %u)"), 0));
+    strUsage += HelpMessageOpt("-verifypowonly", _("Do a PoW-only verification of blocks during initial block download"));
```

Comment:
This PR doesn't implement PoW-only verification, as it is verifying a bunch more properties of the block. A better name might be `-ibdskiptxverification`.

Reply:
Oh, `-ibdskiptxverification` is a good name. I'll use that.